src/Corax/IndexWriter.cs

```diff
@@ -372,6 +371,30 @@ void ExactInsert(ReadOnlySpan<byte> value)
 
                 if (binding.HasSuggestions)
                     AddSuggestions(binding, slice);
+
+                scope?.Dispose();
+            }
+        }
+
+        private static unsafe ByteStringContext<ByteStringMemoryCache>.InternalScope CreateNormalizedTerm(ByteStringContext context, ReadOnlySpan<byte> value, out Slice slice)
+        {
+            ulong hash = 0;
+            int length = value.Length;
+            if (length > Constants.Terms.MaxLength)
+            {
+                int hashStartingPoint = Constants.Terms.MaxLength - 2 * sizeof(ulong);
+                hash = Hashing.XXHash64.Calculate(value.Slice(hashStartingPoint));
+
+                Span<byte> localValue = stackalloc byte[Constants.Terms.MaxLength];
+                value.Slice(0, Constants.Terms.MaxLength).CopyTo(localValue);
```

Comment:
Shouldn't be `Slice(0, hashStartingPoint )` ?

Reply:
Yes, even though effectively there is not difference as we are overwriting the end of the key with the hash anyways.